src/Leprechaun.CodeGen.Roslyn/Scripts/Constants.csx

```diff
@@ -1,48 +1,49 @@
-﻿// Generates Sitecore Habitat-style constants classes
+﻿// Generates Glass Constants File
```

Comment:
I understand why you renamed the original `constants.csx` to `habitat.csx`, but I'm not sure what the purpose of this glass constants file is

Reply:
The reason for the glass mapper specific constant (which I could combine into the main GlassMapper.csx) is to provide the Template Id (as an ID or as a String) and the Field Id's and Names to the attributes in the models.  Without this information, Glass would not always map to the correct field.

Ex.

`[SitecoreField(FieldName` = NameOfFieldFromConstantsFileAsAString)]

Same goes for the SitecoreType attribute which is used on the Class definition.

Reply:
Why not bake it right into the template like the Synthesis one does: https://github.com/blipson89/Leprechaun/blob/master/src/Leprechaun.CodeGen.Roslyn/Scripts/Synthesis.csx#L47

Reply:
I think that makes sense.  I think I got too caught up into separating the code out, but ultimately they will all appear in one file anyways.  So I will make that change later today.